src/Employer/Employer.Web/Orchestrators/DashboardOrchestrator.cs

```diff
@@ -151,5 +169,25 @@ private string GetFilterHeading(int totalVacancies, FilteringOptions filteringOp
                     return $"{totalVacancies} {filterText} {"vacancy".ToQuantity(totalVacancies, ShowQuantityAs.None)}";
             }
         }
+
+        private TransferredVacanciesAlertViewModel GetTransferredVacanciesAlertViewModel(IEnumerable<VacancySummary> vacancies, DateTime? userLastDismissedDate)
```

Comment:
I was thinking ahead a little bit, there are about four types of alerts that we have anticipated. Working out all the different alerts in orchestrator may make this class too big and alerts feels like cross concern to me. I was thinking inline with having a UserAlertManager with methods Get and Dismiss. Perhaps having a single partial view that can be reused when displaying alerts on top of views other then Dashboard. 

This is just an idea, I have not completely thought this through.

Reply:
I was thinking along the same lines.  I'm going to keep it as it is for now until I've implemented a few alerts and then see if its worth pulling out common behaviour.